src/reader/components/ReaderToken.vue

```diff
@@ -25,18 +26,28 @@
 
 <script>
   import { SELECT_TOKEN, MODULE_NS } from '@/reader/constants';
-  import { SELECT_NAMED_ENTITIES } from '@/constants';
+  import { CLEAR_NAMED_ENTITIES, SELECT_NAMED_ENTITIES } from '@/constants';
 
   export default {
     props: ['token'],
     methods: {
       onSelect() {
-        this.$store.dispatch(SELECT_NAMED_ENTITIES, {
-          entities: this.entities,
-        });
-        this.$store.dispatch(`${MODULE_NS}/${SELECT_TOKEN}`, {
-          token: this.token,
-        });
+        if (this.token === this.selectedToken) {
+          this.$store.dispatch(CLEAR_NAMED_ENTITIES);
+          this.$store.dispatch(`${MODULE_NS}/${SELECT_TOKEN}`, {
+            token: null,
+          });
+        } else if (
```

Comment:
@paltman I found a small edge case with the `else if` when in named entities mode

1) Browse to https://deploy-preview-60--explorehomer-dev.netlify.app/reader?urn=urn%3Acts%3AgreekLit%3Atlg0012.tlg001.perseus-grc2%3A1.1-1.7
2) Enabled named entities mode
3) See recording at https://share.getcloudapp.com/z8uZEPRo